ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-master-presubmits.yaml

```diff
@@ -929,6 +929,79 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-azure,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
Given that this is a not very health job, are we ok with making this false for now and running on a per-need basis?

I know it's optional but the MCO repo has quite a lot of always_run tests that are very noisy with failures. I'd prefer if we only mark this to always run once we are satisfied with its stability, such that we can actually use it to catch regressions.

Reply:
yes, no problem. I just updated the PR